pyramid_oereb/core/readers/real_estate.py

```diff
@@ -63,7 +63,8 @@ def read(self, params, nb_ident=None, number=None, egrid=None, geometry=None):
             plan_for_land_register_config.get('layer_opacity'),
             Config.get('default_language'),
             Config.get('srid'),
-            Config.get('proxies')
+            Config.get('proxies'),
+            Config.get('verify_certificate')
```

Comment:
plan_for_land_register_config.get('verify_certificate', True) ?

Reply:
You are correct, I have now adapted the code